enhancements/ingress/global-admission-hsts.md

```diff
@@ -0,0 +1,568 @@
+---
+title: global-options-enable-hsts 
+authors:
+  - "@cholman"
+
+reviewers:
+  - "@danehans" 
+  - "@frobware"
+  - "@knobunc"
+  - "@Miciah"
+  - "@miheer"
+  - "@rfredette"
+  - "@sgreene570"
+
+approvers:
+  - "@frobware"
+  - "@knobunc"
+
+creation-date: 2021-04-16
+
+last-updated: 2021-04-20
+
+status: implementable
+
+see-also:
+  - https://bugzilla.redhat.com/show_bug.cgi?id=1512759
+  - https://bugzilla.redhat.com/show_bug.cgi?id=1430035
+
+replaces:
+  - enhancements/enhancements/ingress/global-options-enable-hsts.md
+
+superseded-by:
+
+---
+
+# Global Admission Plugin for HTTP Strict Transport Security (HSTS)
+
+This enhancement provides the capability to enforce global automatic HSTS
+for TLS routes.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+In 3.x and 4.x customers can [provide a per-route annotation to enable HSTS](
+https://docs.openshift.com/container-platform/4.4/networking/routes/route-configuration.html#nw-enabling-hsts_route-configuration).
+For customers with many routes or regulatory compliance issues, the manual per-route annotation is seen as sub-optimal.
+
+This enhancement extends the `Ingress.config.openshift.io` API and adds a new `route` admission controller to the OpenShift API server
+which together allow cluster administrators to enforce HSTS globally. This enhancement also provides a recommendation for batch route annotation configuration.
```

Comment:
> This enhancement also provides a recommendation for batch route annotation configuration.

Is this included in the enhancement and I just missed it?

Reply:
Thanks.  I just pushed an update for that.